analytics/java/src/main/java/com/koverse/examples/analytics/SentimentAnalysis.java

```diff
@@ -35,13 +36,15 @@
             .displayName("Text field")
             .parameterName(TEXT_COL_PARAM)
             .required(Boolean.TRUE)
-            .type(Parameter.TYPE_COLLECTION_FIELD)
+            .type(Parameter.TYPE_STRING)
```

Comment:
Is this right? TYPE_COLLECTION_FIELD should be what is used when you want a user to choose a field name that exists in one of the input data sets. It will show up as a string in the context parameter values when the transform is run

Reply:
I will try it again.